packages/stryker/src/ConfigValidator.ts

```diff
@@ -34,6 +34,21 @@ export default class ConfigValidator {
     }
   }
 
+  private validateMutator() {
+    const mutator = this.strykerConfig.mutator;
+    if (typeof mutator !== 'string' && !_.isObject(mutator)) {
+      // console.log(`***** FAILED at first assertion: Mutator is: ${mutator}`);
+      this.invalidate(`Value "${mutator}" is invalid for \`mutator\`. Expected either a string or an object`);
+    }
+    if (_.isObject(mutator)) {
```

Comment:
Please use `typeof mutator === 'object'` instead. We don't want to depend on lodash for these simple assertions.